velox/functions/sparksql/Bitwise.cpp

```diff
@@ -33,9 +33,59 @@ struct BitwiseOrFunction {
   }
 };
 
+template <typename T>
+struct ShiftLeftFunction {
+  template <typename TInput1, typename TInput2>
+  FOLLY_ALWAYS_INLINE void call(TInput1& result, TInput1 a, TInput2 b) {
+    if constexpr (std::is_same_v<TInput1, int32_t>) {
+      if (b < 0)
+        b = b % 32 + 32;
```

Comment:
nit: Can you add '{' , Its required as per our coding standards.

Reply:
Done.
Thanks!